plugins/catalog-backend-module-system-entity-model/README.md

```diff
@@ -0,0 +1,3 @@
+# @backstage/plugin-catalog-backend-module-system-entity-model
+
+Implements the [default System Entity Model](https://backstage.io/docs/features/software-catalog/descriptor-format) for the catalog backend.
```

Comment:
Maybe include instructions on installing and customizing this module (if any)?

Reply:
ah yeah i should probably do that, will do in a moment

Reply:
Hm I changed my mind. Given that it's "installed" (used by the catalog) by default right now, and since there's some discussions whether this should be exposed as a module at all before a real model extension is invented ... maybe it's best to not tell people to install it explicitly yet